node/states.js

```diff
@@ -42,29 +42,21 @@ module.exports.LockedUnhealthyState = LockedUnhealthyState;
  * - channel.random
  */
 
-var symptoms = {
```

Comment:
Ohh I see, this is where that was tucked away before? Well, errors module is a huge step up at least ;-)